portfolio/src/main/webapp/Home.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="Home.css">
+	<link rel="stylesheet" href="General.css">
```

Comment:
Maybe General.css should be named common.css?
Also I think the convention is to use lowercase filenames for CSS files

Reply:
fixed